src/ManageCourses.Api/Startup.cs

```diff
@@ -51,9 +53,15 @@ public void ConfigureServices(IServiceCollection services)
                 .AddDbContext<ManageCoursesDbContext>(
                     options =>
                     {
+                        const int maxRetryCount = 3;
+                        const int maxRetryDelaySeconds = 5;
+
+                        var postgresErrorCodesToConsiderTransient = new List<string>(); // ref: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/blob/16c8d07368cb92e10010b646098b562ecd5815d6/src/EFCore.PG/NpgsqlRetryingExecutionStrategy.cs#L99
```

Comment:
is this needed?

Reply:
yes it doesn't have an overload without that param <sadface>